java-test-samples/async-lambda-dynamodb/README.md

```diff
@@ -0,0 +1,45 @@
+# App
+
+This project contains an AWS Lambda maven application with [AWS Java SDK 2.x](https://github.com/aws/aws-sdk-java-v2) dependencies.
```

Comment:
Detail explanation is required as to what is the behavior of the app. Providing architecture diagram will be really helpful.